packages/tupaia-web-server/src/routes/EntitiesRoute.ts

```diff
@@ -0,0 +1,45 @@
+/**
+ * Tupaia
+ * Copyright (c) 2017 - 2023 Beyond Essential Systems Pty Ltd
+ */
+
+import { Request } from 'express';
+import { Route } from '@tupaia/server-boilerplate';
+import { Entity } from '@tupaia/types';
+
+export type EntitiesRequest = Request<
+  any,
+  any,
+  any,
+  any
+>;
```

Comment:
We have a whole ticket dedicated to that WAITP-1278, I was planning to define these types a bit better during that update (since we're still making small updates and adjustments to the specifics of the API as we continue the rewrite). But happy to define it now if you insist 🙏

Reply:
Oh nah, that's all good then 👍